exporters/sender/jdk/src/main/java/io/opentelemetry/exporter/sender/jdk/internal/JdkHttpSender.java

```diff
@@ -120,8 +118,7 @@ private static HttpClient configureClient(
       builder.sslContext(sslContext);
     }
     if (proxyOptions != null) {
-      builder.proxy(
-          ProxySelector.of(new InetSocketAddress(proxyOptions.getHost(), proxyOptions.getPort())));
+      builder.proxy(proxyOptions.getProxySelector());
```

Comment:
it's nice to see how we can just pass the ProxySelector down, helps confirm this approach 👍